Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,173 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/grid_2"
+      android:layout_marginEnd="@dimen/grid_2"
+      android:layout_marginHorizontal="@dimen/grid_0_5"
+      android:layout_marginVertical="@dimen/grid_4">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/email_app_bar_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/grid_2"
+        android:paddingBottom="@dimen/grid_2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+      <com.google.android.material.appbar.MaterialToolbar
+          android:id="@+id/toolbar"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          app:title="@{email.subject}"
+          app:titleTextAppearance="?attr/textAppearanceBodyExtraLarge"
+          app:titleCentered="true"
+          app:subtitle="3 more messages"
```

Comment:
Move to field in Email?

Reply:
Yes. I have to update the Email data object.